roles/openshift_logging_fluentd/defaults/main.yml

```diff
@@ -55,4 +56,4 @@ openshift_logging_fluentd_aggregating_passphrase: none
 #fluentd_throttle_contents:
 #fluentd_secureforward_contents:
 
-openshift_logging_fluentd_file_buffer_limit: 1Gi
+openshift_logging_fluentd_file_buffer_limit: 256Mi
```

Comment:
Note that we keep the file buffer limit small, one half of the memory limit by default.